packages/connect-web/src/connectSettings.ts

```diff
@@ -24,6 +24,19 @@ export const getEnv = () => {
 
 declare let global: any;
 
+const processQueryString = (url: string, keys: string[]) => {
+    const vars = url.replace("?", '').split('&');
```

Comment:
Maybe it would be nice to use  `URLSearchParams` the built-in  function for using querystring, like:

```
    const searchParams = new URLSearchParams(url);
    const result = {};
    for (const [key, value] of searchParams.entries()) {
        if (keys.includes(key)) {
            result[key] = decodeURIComponent(value);
        }
    }
    return result;
```

Reply:
ok why not 582ea3aea7623e790ce793e6f45df3e68c0b5fb6

Reply:
hmm this triggers lint error: "error  iterators/generators require regenerator-runtime" I'd remove it if you dont mind. or is there a way how to work with iterators without this?

Reply:
I think it should be possible to transform the iterator to an array like: 

```
 const paramArray = Array.from(searchParams.entries());
```

Reply:
And then user forEach normally.